index.mako

```diff
@@ -100,9 +100,9 @@ conda install -c conda-forge pre-commit
 pre-commit --version
 ```
 
-### 2. Add a pre-commit configuration
+### 2. Add a pre-commit configuration to your project
 
-- create a file named `.pre-commit-config.yaml`
+- create a file named `.pre-commit-config.yaml` in the root level of your project
```

Comment:
this is already mentioned [here](https://pre-commit.com/#adding-pre-commit-plugins-to-your-project)